### ISSUE TITLE
Add Group Skipping to A11Y Test Docs

### ISSUE THREAD

**Issue by Zeyber** [SAP/cloud-commerce-spartacus-storefront#6518]
Group Skipping is not a part of writing A11Y Tests where it should be included. Need to add info about it to: https://wiki.hybris.com/display/spar/How+to+implement+accessibility+tests+and+fixes

**Comment by Zeyber** [comment]
Group skipping section added to wiki
